### DISCUSSION TITLE
Holiday Hours for Meetings

### DISCUSSION THREAD

**Discussion by D51WebAdmin** [code4recovery/12-step-meeting-list#1298] [Ideas]
Hello Everyone,

I'm pretty novice when it comes to web dev so there might be solutions for this already but please bare with me. I had a request to mark a meeting as "cancelled" on Christmas. I don't think there is a way to currently do this. I am envisioning the "Meeting Information" section to have a fill-in-the-blank for holidays with a calendar option next to it. That way we could mark the meeting as "temporarily cancelled due to holiday". That could also be a cool search criteria for those looking for active meetings during the holiday from a user experience point of view. Any ideas if this is something that is already being worked on? If I knew how to code I would help implement this but unfortunately I don't. Thank you in advanced for any help or work-around that is offered.

**Comment by kiyote33** [comment]
Hello @D51WebAdmin, a suggestion to achieve what you ask might be to set the Location Information radio buttons to No where it asks if you can attend the meeting in person. That will cause "inactive" text to appear on the meeting list next to the meeting name. You may want to insert additional text in your "notes" field stating when the meeting will return to being active. Additional task will be to reset the attendance option to Yes in the week before the next active meeting.  Hope this helps...
![image](https://github.com/code4recovery/12-step-meeting-list/assets/57125427/c323a5ef-8453-428e-a797-ecbe45fae27e)

**Comment by joshreisner** [comment]
this won't be good as a short term fix, but we do have a longer-term idea on how to modify the spec to accomplish this: https://github.com/code4recovery/spec/discussions/80#discussioncomment-7935313
